routes.yaml

```diff
@@ -3,4 +3,5 @@ group: api
 
 weight: 90
 routes:
-  1: ^/company/(.*)/auth-code/*
\ No newline at end of file
+  1: ^/company/(.*)/auth-code/*
+  2: ^/auth-code/healthcheck
```

Comment:
Just wondering if this should be `^/auth-code-notification/healthcheck` , since that's the name of the service? I know this doesn't match what we did for OE API but perhaps that is wrong too.

If yes, change would also apply to the application.properties file and the Docker config PR.

Reply:
yes that might be better as it is the healthcheck of the service and the auth-code is the restful url pattern

Reply:
We should go back and change that for OE API too. I'll check first with FESS in case the existing health-check URL there is already being used (by some kind of monitoring).

Reply:
Thanks.